src/System.Collections.NonGeneric/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,11 @@
+TypesMustExist : Type 'System.Collections.ArrayList' does not exist in the implementation but it does exist in the contract.
```

Comment:
None of these make sense.  Are you sure you are passing in the targeting pack reference assemblies?